### PR TITLE
Use automatic args erase feature from modelforge logging

### PR DESCRIPTION
**Pull request by irinakhismatullina**
Use new `modelforge.slogging.add_logging_args()` parameter: `erase_args=True`

### CODE REVIEW

**Comment by vmarkovtsev** [comment]
Build failure

**Comment by irinakhismatullina** [comment]
@vmarkovtsev This happens because the code now is incompatible with latest changes in `lookout-sdk-ml`. So should I use the old version, and explicitly demand new version for `modelforge`?

**Comment by vmarkovtsev** [comment]
@irinakhismatullina According to the CI logs, the reason is different, please fix the tests.

**Comment by vmarkovtsev** [comment]
You can hack `.travis.yml` and `Dockerfile` to force the right packages anyway

**Comment by irinakhismatullina** [comment]
Well this is definitely related to the newest changes in `lookout-sdk-ml`: https://travis-ci.com/src-d/style-analyzer/jobs/186382553#L1427
And this was also changed in the last release https://travis-ci.com/src-d/style-analyzer/jobs/186382553#L1435

The question is: do i need roll back to `lookout-sdk-ml==0.15.0` as it were before (because clearly no work was done through the format (and maybe typos too) analyzer to support the updated?

**Comment by irinakhismatullina** [comment]
The problem [here](https://travis-ci.com/src-d/style-analyzer/jobs/187393949#L1051) is `sourced-ml` requirements for `modelforge<12.0`. The same happens during local default package installation. So what's the best way to deal with this situation @vmarkovtsev ? Add `modelforge==12.0` specifically to the requirements?

**Comment by zurk** [comment]
I close this one because it is being taken over by https://github.com/src-d/style-analyzer/pull/719

**Comment by vmarkovtsev** [comment]
@zurk the second case is not covered in 719

**Comment by vmarkovtsev** [comment]
@irinakhismatullina Can you please rebase

**Comment by zurk** [comment]
One test in docker is failed:
```
======================================================================
ERROR: test_handlers (lookout.style.format.tests.test_main.MainTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/style-analyzer/lookout/style/format/tests/test_main.py", line 37, in test_handlers
    parser = main.create_parser()
  File "/style-analyzer/lookout/style/format/cmdline_tools.py", line 104, in create_parser
    slogging.add_logging_args(parser, erase_args=True)
TypeError: add_logging_args() got an unexpected keyword argument 'erase_args'
----------------------------------------------------------------------
```
because of wrong modelforge version. 
Version conflict fun continues.

File: lookout/style/format/cmdline_tools.py [hunk 1]
```diff
@@ -101,8 +101,7 @@ def create_parser() -> ArgumentParser:
     parser = ArgumentParser(formatter_class=ArgumentDefaultsHelpFormatterNoNone)
 
     # General options
-    slogging.add_logging_args(parser)
-
+    slogging.add_logging_args(parser, erase_args=True)
```
Comment (vmarkovtsev) [inline lookout/style/format/cmdline_tools.py]: It is True by default

File: lookout/style/typos/cmdline_tools.py [hunk 2]
```diff
@@ -80,7 +80,7 @@ def create_parser() -> ArgumentParser:
     parser = ArgumentParser(formatter_class=ArgumentDefaultsHelpFormatterNoNone)
 
     # General options
-    slogging.add_logging_args(parser)
+    slogging.add_logging_args(parser, erase_args=True)
```
Comment (vmarkovtsev) [inline lookout/style/typos/cmdline_tools.py]: Same here

**Comment by vmarkovtsev** [comment]
@zurk I know why. Because

```
Installing collected packages: lookout-style, modelforge, pyspark
  Running setup.py develop for lookout-style
  Found existing installation: modelforge 0.12.0
    Uninstalling modelforge-0.12.0:
      Successfully uninstalled modelforge-0.12.0
Successfully installed lookout-style modelforge-0.11.1 pyspark-2.2.1
```

👿 🔥 😠

**Comment by vmarkovtsev** [comment]
Should be fixed in https://github.com/src-d/style-analyzer/pull/727

**Comment by vmarkovtsev** [comment]
The tests have passed, but not for the right reason hehe

**Pull request merged**